test/e2e/bound_sa_token_test.go

```diff
@@ -237,7 +238,7 @@ func TestChangeServiceAccountIssuer(t *testing.T) {
 	// Wait for operator readiness
 	test.WaitForKubeAPIServerClusterOperatorAvailableNotProgressingNotDegraded(t, configClient)
 
-	defaultIssuer := "auth.openshift.io"
+	defaultIssuer := "https://apiserver.openshift-kube-apiserver.svc"
```

Comment:
In order for the SA OIDC discovery to work, this must be reachable from the outside of the cluster

Reply:
/hold
see https://github.com/openshift/api/pull/617#discussion_r404755006 first, since this PR somewhat depends on the api, I think we should add the hold to this PR until the discussion there is resolved

Reply:
As per the above discussion, set to `https://kubernetes.default.svc` please

Reply:
Updated, PTAL.